File: Student.java
//student is the class
public class Student {
private  String name ; //name is a variable
private int age ; //age is a variable
public Student (String names , int ages ){ //constructor--> initialize the variables and create the object
    name = names;
    age = ages;
}
public void display() { //function
    System.out.println("name" + name + "age" + age);
}
}

File: student.py
class Student: #class
    def __init__(self,names,ages): #constructor
        self.name=names  #variables
        self.age=ages
    def display(self): #function
        print("name",self.name,"age",self.age)

#class creates objects
#objects variables need to be initlialized
# through objects we can invoke functions  
st=Student("amrita",15)  
st.display()


